dspace-api/src/main/java/org/dspace/app/mediafilter/PDFBoxThumbnail.java

```diff
@@ -77,7 +77,7 @@ public InputStream getDestinationStream(Item currentItem, InputStream source, bo
 
     @Override
     public String[] getInputMIMETypes() {
-        return ImageIO.getReaderMIMETypes();
+        return new String[]{"Adobe PDF"};
```

Comment:
I wonder if we should instead simply update this class to remove the `implements SelfRegisterInputFormats`.  It seems like the three `getInput*()` methods here are not really doing anything useful.  Most other Filters don't even include that `implements`.

By the way, this PR currently fails in Travis cause you are still importing `ImageIO` into this class even though it's no longer used.